src/cljc/athens/common_events.cljc

```diff
@@ -203,6 +203,19 @@
                      :value value}}))
 
 
+(defn build-unindent-multi-event
+  "Builds `:datascript/unindent-multi` event with:
+  - `uids` : `:block/uid` of selected blocks
+  - `f-uid`: uid after passing through `uid-and-embed-id` function"
```

Comment:
when would we expect it not to be same `uid`?

Reply:
The description I wrote here is wrong, this is `first-uid` from the `uids`
```
[f-uid f-embed-id]          (common-db/uid-and-embed-id (first uids))
```

Reply:
ok, but still "when do we expect `f-uid` to be different from `(first uids)`?"

Reply:
Lets say we have 2 blocks:
- block 1 which has a `block embed` 
- block 2 with text
```
 - parent block
   - block 1 _Some embeded block_
   - block 2
```

Now here under `parent block` if we select the `embed block` and `block 2` we will have `embed block` uid as the first in `uids`. An example of embed block uid is `"e5dec8a28-embed-386e7535-fed0-4c8d-b922-99eb3ce43ec2"` which when passed through `uid-and-embed-id` outputs `[ e5dec8a28  386e7535-fed0-4c8d-b922-99eb3ce43ec2]`

Reply:
`embed block` is client only concept.
`f-uid` -> `block-uid` since it really is `:block/uid`
and let's call `e5dec8a28-embed-386e7535-fed0-4c8d-b922-99eb3ce43ec2` something different then `uid`